make/photon/chartserver/Dockerfile

```diff
@@ -1,10 +1,25 @@
 FROM photon:2.0
 
+COPY ./make/photon/tdnf/photon.internal ./make/photon/tdnf/photon-updates.internal /etc/yum.repos.d/
+
+# use the internal tdnf repo server
+ARG USE_INTERNAL_TDNF
```

Comment:
We don't need this flag, we can check if the photon.internal and photon-updates.internal is updated or not.